apps/backend/user/src/verifyMFA.ts

```diff
@@ -0,0 +1,18 @@
+import { User } from "./models/user";
+import { TOTP } from "otpauth";
+
+const verifyMFA = async (user: User, totp_secret: string) => {
+  // Create TOTP from stored secret
+  const totp = new TOTP({
+    label: "Martletplace",
```

Comment:
```suggestion
    label: "martletplace",
```

Reply:
I think this is only important for the CreateAccount.tsx appearance of this, but that exact string is displayed as the label in auth apps so it makes sense to keep the capitalization of "MartletPlace". It should still be made consistent however.